[2024-02-29T17:26:08.486+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dataIngestionDag.push_csv_to_s3 manual__2024-02-29T17:25:50.528999+00:00 [queued]>
[2024-02-29T17:26:08.492+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dataIngestionDag.push_csv_to_s3 manual__2024-02-29T17:25:50.528999+00:00 [queued]>
[2024-02-29T17:26:08.493+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 3
[2024-02-29T17:26:08.508+0000] {taskinstance.py:2214} INFO - Executing <Task(PythonOperator): push_csv_to_s3> on 2024-02-29 17:25:50.528999+00:00
[2024-02-29T17:26:08.513+0000] {standard_task_runner.py:60} INFO - Started process 31480 to run task
[2024-02-29T17:26:08.517+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'dataIngestionDag', 'push_csv_to_s3', 'manual__2024-02-29T17:25:50.528999+00:00', '--job-id', '149', '--raw', '--subdir', 'DAGS_FOLDER/ingestionPipeline.py', '--cfg-path', '/tmp/tmptmq2yt5n']
[2024-02-29T17:26:08.519+0000] {standard_task_runner.py:88} INFO - Job 149: Subtask push_csv_to_s3
[2024-02-29T17:26:08.554+0000] {task_command.py:423} INFO - Running <TaskInstance: dataIngestionDag.push_csv_to_s3 manual__2024-02-29T17:25:50.528999+00:00 [running]> on host ip-172-31-12-93.ap-south-1.compute.internal
[2024-02-29T17:26:08.629+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='shubh.sehgal2506@gmail.com' AIRFLOW_CTX_DAG_OWNER='shubh-airflow' AIRFLOW_CTX_DAG_ID='dataIngestionDag' AIRFLOW_CTX_TASK_ID='push_csv_to_s3' AIRFLOW_CTX_EXECUTION_DATE='2024-02-29T17:25:50.528999+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-02-29T17:25:50.528999+00:00'
[2024-02-29T17:26:08.935+0000] {python.py:202} INFO - Done. Returned value was: CSV FILE PUSHED TO S3
[2024-02-29T17:26:08.951+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=dataIngestionDag, task_id=push_csv_to_s3, execution_date=20240229T172550, start_date=20240229T172608, end_date=20240229T172608
[2024-02-29T17:26:08.970+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-29T17:26:08.986+0000] {taskinstance.py:3309} INFO - 1 downstream tasks scheduled from follow-on schedule check
